src/components/Property/Property.js

```diff
@@ -69,14 +77,13 @@ export default class Property extends PureComponent {
           {isClickable && <Indicator className={classes.indicator} direction={indicatorDirection} />}
         </td>
         <td className={classes.info}>
-          {title && <span className={classes.title}>{title}</span>}
```

Comment:
Title is part of JSON Schema and supported by the spec, https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/versions/3.0.md#schema-object